helpers/pagination/paginate.py

```diff
@@ -48,6 +49,12 @@ def resolve_has_previous(self, has_previous):
 
         return has_previous
 
+    def resolve_current_page(self, pages):
+        pages = self.resolve_pages(pages)
+        if self.page > pages:
+            return pages
+        return self.page
```

Comment:
I see you return the total number of pages as the current page when a user sends a query with a page number that doesn't exist. Won't it be better to return a null or false instead for the current page since it is not actually returning a valid page?
Nice work though @nakatuddesuzan.

Reply:
If I understand correctly this works such that when the page passed is greater than the last page then just send them to the last page.

Reply:
Yes @kampkelly , What @caveinn  has pointed is the logic I used.

Reply:
The user is sent to the last page. The last page == total number of pages. So the page being returned is valid.

Reply:
Okay.

Reply:
If there are no more resources, do we need to return the last page of the resources as the current page? For instance, would this mean that the page 100 == page 7 as in the screenshot above?

Reply:
If there are no more resources, do we need to return the last page of the resources as the current page? For instance, would this mean that the page 100 == page 7 as in the screenshot above?

Reply:
Okay, I now get the argument.  Let me return null for the current page. @Bonifase  and @kampkelly

Reply:
This feedback has been worked on @Bonifase